test/data/parameters_template.yaml

```diff
@@ -0,0 +1,8 @@
+{{ namespace }}:
+  names:
+    {% for robot in robots %}
+      {{ robot.name }}: "{{ robot.ip }}"
+    {% endfor %}
+radians: {{ radians(120) }}
```

Comment:
And are these parsed equivalently to !degrees/!radians?

Reply:
Yes, plus with jinja we could do more mathematical operations